tests-invoke

```diff
@@ -19,257 +19,82 @@
-        try:
-            del sink.status["extras"]
-        except KeyError:
-            pass
-        sink.flush()
-
-        return ret
-
-    def run(self, opts):
-        # Collision detection
-        ret = self.detect_collisions(opts, False)
-        if ret:
-            sys.stderr.write('Collision detected: {0}\n'.format(ret))
-            return None
-
-        if opts.publish:
-            self.start_publishing(opts.publish)
-            os.environ["TEST_ATTACHMENTS"] = self.sink.attachments
-
-        os.environ["TEST_NAME"] = self.name
+        return None
 
+    try:
         # Split OS and potential scenario
-        (image, _, scenario) = self.context.partition("/")
+        (image, _, scenario) = opts.context.partition("/")
+
+        # Set up tests environment variables
         if scenario:
             os.environ["TEST_SCENARIO"] = scenario
         os.environ["TEST_OS"] = image
 
-        msg = "Testing {0} for {1} with {2} on {3}...\n".format(self.revision, self.name,
-                                                                self.github_context, HOSTNAME)
-        sys.stderr.write(msg)
-
-        ret = None
-
-        # Flush our own output before running command
-        sys.stdout.flush()
-        sys.stderr.flush()
+        # Each project prepares bots for itself. If specific ref exists, set it up.
+        if opts.bots_ref:
+            os.environ["COCKPIT_BOTS_REF"] = opts.bots_ref
 
         # Actually run the tests
-        if not ret:
-            entry_point = os.path.join(BASE_DIR, ".cockpit-ci", "run")
-            alt_entry_point = os.path.join(BASE_DIR, "test", "run")
-            if not os.path.exists(entry_point) and os.path.exists(alt_entry_point):
-                entry_point = alt_entry_point
-            cmd = ["timeout", "120m", entry_point]
-            if opts.verbose:
-                cmd.append("--verbose")
-            p = subprocess.Popen(cmd)
-            while p.poll() is None:
-                # Cancel test if PR gets updated while running
-                ret = self.detect_collisions(opts, True)
-                if ret:
-                    sys.stderr.write('Collision detected, cancelling this test run: {0}\n'.format(ret))
-                    ret = None
-                    p.terminate()
-                    p.wait(timeout=60)
-                else:
-                    time.sleep(60)
-            ret = p.returncode
-            sys.stderr.write("Test run finished, return code: {0}\n".format(ret))
-        # All done
-        if self.sink:
-            ret = self.stop_publishing(ret)
+        entry_point = os.path.join(BASE_DIR, ".cockpit-ci", "run")
+        alt_entry_point = os.path.join(BASE_DIR, "test", "run")
+        if not os.path.exists(entry_point) and os.path.exists(alt_entry_point):
+            entry_point = alt_entry_point
+        cmd = ["timeout", "120m", entry_point]
+        if opts.verbose:
+            cmd.append("--verbose")
```

Comment:
I know this isn't new code, but none of our projects actually supports this. So this and the whole `--verbose` option could be dropped.